### PR TITLE
LINK-336 | Add Matomo tracking to the project and enable it on staging environment

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Add Matomo tracking to the project and enable it on staging environment

## Closes
[LINK-336](https://helsinkisolutionoffice.atlassian.net/browse/LINK-336)

### CODE REVIEW

File: src/domain/app/layout/PageWrapper.tsx [hunk 1]
```diff
@@ -31,6 +38,16 @@ const PageWrapper: React.FC<PageWrapperProps> = ({
...
 
+  // Track page view
+  useDeepCompareEffect(() => {
+    if (consent.tracking) {
+      trackPageView({
+        documentTitle: translatedTitle,
+        href: window.location.href,
+      });
+    }
+  }, [{ pathname: location.pathname, search: location.search }]);
```
Comment (Sartonon) [inline src/domain/app/layout/PageWrapper.tsx:49]: is useDeepCompareEffect needed here?

Comment (Sartonon) [reply src/domain/app/layout/PageWrapper.tsx:49]: Does this have exactly the same behaviour?
```
React.useEffect(() => {}, [location.pathname, location.search])
```

**Review by Sartonon — COMMENTED** [review]
LGTM :) One comment about deepCompare

File: src/domain/app/layout/PageWrapper.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/app/layout/PageWrapper.tsx]: You are absolutely right.  Changed it to 
```
React.useEffect(() => {
    if (consent.tracking) {
      trackPageView({
        documentTitle: translatedTitle,
        href: window.location.href,
      });
    }
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [consent.tracking, location.pathname, location.search]);
  ```
  so the page is tracked also after accepting cookies

**Pull request merged**